scrapy/exporters.py

```diff
@@ -139,8 +139,9 @@ def _export_xml_field(self, name, serialized_value):
         if hasattr(serialized_value, 'items'):
             for subname, value in serialized_value.items():
                 self._export_xml_field(subname, value)
-        elif hasattr(serialized_value, '__iter__'):
-            for value in serialized_value:
+        elif (hasattr(serialized_value, '__iter__')
+              and not isinstance(serialized_value, six.string_types)):
+              for value in serialized_value:
```

Comment:
Is it possible to use https://github.com/scrapy/scrapy/blob/e0074d7f4d3a7fcff3f3f00b4db7175b7fdf2fba/scrapy/utils/python.py#L47 here?

Also, indentation looks off :)